miniapps/nurbs/nurbs_ex1p.cpp

```diff
@@ -25,6 +25,8 @@
 //               mpirun -np 4 nurbs_ex1p -m ../../data/square-nurbs.mesh -o 2 -no-ibp
 //               mpirun -np 4 nurbs_ex1p -m ../../data/cube-nurbs.mesh -o 2 -no-ibp
 //               mpirun -np 4 nurbs_ex1p -m ../../data/pipe-nurbs-2d.mesh -o 2 -no-ibp
+//               mpirun -np 4 nurbs_ex1p -m ../../../miniapps/nurbs/meshes/square-nurbs.mesh -r 4 -pm "1" -ps "2"
```

Comment:
```suggestion
//               mpirun -np 4 nurbs_ex1p -m meshes/square-nurbs.mesh -r 4 -pm "1" -ps "2"
```

Reply:
I am afraid that this would not work as the directory `miniapps/nurbs/meshes/` is not copied from source to binary. So unfortunately we have to find the mesh in the source folder.